src/validator.directive.js

```diff
@@ -239,9 +239,9 @@
         /**
          * Set initial validity to undefined if no boolean value is transmitted
          */
-        var initialValidity;
-        if (typeof scope.initialValidity === 'boolean') {
-          initialValidity = scope.initialValidity;
+        var initialValidity = void 0;
```

Comment:
void 0 is the best way to initialize any variable in Javascript. What it actually does is evaluating the expression and returning undefined.